paasta_tools/setup_kubernetes_job.py

```diff
@@ -233,6 +233,8 @@ def setup_kube_deployments(
     eks: bool = False,
 ) -> bool:
 
+    existing_apps = set()
+    existing_kube_deployments = set()
```

Comment:
mypy is complaining about these not having type definitions.

Reply:
oh Pylance was complaining about not being defined , maybe I forgot to re run mypy after changing that